src/System.Collections.NonGeneric/ref/System.Collections.NonGeneric.cs

```diff
@@ -71,6 +71,15 @@ public partial class CaseInsensitiveComparer : System.Collections.IComparer
         public static System.Collections.CaseInsensitiveComparer DefaultInvariant { get { return default(System.Collections.CaseInsensitiveComparer); } }
         public int Compare(object a, object b) { return default(int); }
     }
+    [System.ObsoleteAttribute("Please use StringComparer instead.")] 
```

Comment:
We decided to not add any new [Obsolete] attributes, but it looks like you're just bringing over ones that already exist.

Incidentally I'm not sure what it would mean to add new obsoletes here on the contract/façade - presumably it wouldn't break anyone referencing mscorlib directly, and of course nobody referencing the contract if it's a newly added member.